### PR TITLE
adding System.Reactive.Core to fix proxies

### PR DESCRIPTION
**Pull request by brettsam**
All tests passed previously b/c the package was referenced in test projects, but not in the actual product. Moving package ref.

### CODE REVIEW

**Review by soninaren — APPROVED** [review]

File: test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj [hunk 1]
```diff
@@ -158,7 +157,7 @@
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\requirements_PSGalleryOffline.psd1" />
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\requirements_PSGalleryOnline.psd1" />
     <EmbeddedResource Include="Resources\FileProvisioning\PowerShell\profile.ps1">
-    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
```
Comment (fabiocav) [inline test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj:160]: nit: formatting issue here.

**Review by fabiocav — APPROVED** [review]

File: test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj (hunk 1 above)
Comment (brettsam) [reply test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj:160]: yeah, a previous formatting issue that is now fixed! (it's meant to be indented) :-)

**Comment by brettsam** [comment]
Recording depsjson failure here for future reference:

```
Previous file: D:\a_work\1\s\test\WebJobs.Script.Tests\bin\Debug\net6.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json
New file:      D:\a_work\1\s\src\WebJobs.Script.WebHost\bin\Debug\net6.0\Microsoft.Azure.WebJobs.Script.WebHost.deps.json

Changed:

Removed:

Added:
- System.Reactive.Core.dll: 3.0.6000.0/3.0.6000.0

Expected: True
Actual:   False
```

**Review by fabiocav — APPROVED** [review]

**Pull request merged**